web/libs/editor/src/components/Timeline/Controls/ConfigControl.tsx

```diff
@@ -1,19 +1,176 @@
 import type React from "react";
-import { type FC, type MouseEvent, useContext, useEffect, useState } from "react";
-import { Toggle } from "@humansignal/ui";
-import { Block, Elem } from "../../../utils/bem";
-
-import { IconConfig } from "@humansignal/ui";
-import { TimelineContext } from "../Context";
-import { ControlButton } from "../Controls";
-import { Slider } from "./Slider";
+import {type FC, type MouseEvent, useContext, useEffect, useMemo, useRef, useState} from "react";
+import { createPortal } from "react-dom";
+import {Toggle, Tooltip} from "@humansignal/ui";
+import {Block, Elem} from "../../../utils/bem";
+import {Slider as AntSlider, Select} from "antd";
```

Comment:
```suggestion
import {Slider as AntSlider } from "antd";
```
I noticed additional usage of AntD components, we are actively removing our dependence on these in the project, so we can use a Select component available from our internal ui lib. See the below suggestion.